src/test/java/com/hazelcast/aws/AwsDiscoveryStrategyTest.java

```diff
@@ -135,4 +133,32 @@ public void discoverNodesOneAddressOnePort()
         assertEquals(new Address(publicAddress, port), node.getPublicAddress());
         assertFalse(iterator.hasNext());
     }
+
+    @Test
+    public void validateValidRegion() {
+        awsDiscoveryStrategy.validateRegion("us-west-1");
+        awsDiscoveryStrategy.validateRegion("us-gov-east-1");
+    }
+
+    @Test
+    public void validateInvalidRegion() {
+        String region = "us-wrong-1";
```

Comment:
Nit: Could you add the `// given // when // then` to keep the structure of other tests.